qemu/tests/cfg/numa_opts.cfg

```diff
@@ -38,6 +38,11 @@
           numa_node1_size = 128
           ppc64,ppc64le:
               numa_node1_size = 768
+          Host_RHEL.m7.u3:
```

Comment:
Creating restrictions for predecessors is better than this implementation, so that you won't need to append successors (e.g. `m7.u4`) to this list if the numa topology doesn't being changed.